gpMgmt/test/behave/mgmt_utils/gpinitsystem.feature

```diff
@@ -17,6 +17,28 @@ Feature: gpinitsystem tests
         And gpconfig should print "Master  value: off" to stdout
         And gpconfig should print "Segment value: off" to stdout
 
+    Scenario: gpinitsystem creates a cluster when the user set LC_ALL env variable
+        Given create demo cluster config
+        And the user runs command "export LC_ALL=en_US.UTF-8"
+        When the user runs command "gpinitsystem -a -c ../gpAux/gpdemo/clusterConfigFile"
+        Then gpinitsystem should return a return code of 0
+        Given the user runs "gpstate"
+        Then gpstate should return a return code of 0
```

Comment:
This test case is not relevant for 6X because `gpinitsystem` [sets](https://github.com/arenadata/gpdb/blob/ba91667ba5698367d057b26c392447f6db72fbd2/gpMgmt/bin/gpinitsystem#L303) default `en_EN.UTF-8` locale when another value was not specified in command line argument. Also it makes sense to fix help documentation entry for command line [option](https://github.com/arenadata/gpdb/blob/ba91667ba5698367d057b26c392447f6db72fbd2/gpMgmt/doc/gpinitsystem_help#L114-L119) `--locale` according to corresponding [xml doc entry](https://github.com/arenadata/gpdb/blob/ba91667ba5698367d057b26c392447f6db72fbd2/gpdb-doc/dita/utility_guide/ref/gpinitsystem.xml#L146-L158) to not confuse end users.

Reply:
@At1ass please, make required fix to help documentation of `gpinitsystem`